src/components/SpruceForm/Widgets/LeafyGreenWidgets.tsx

```diff
@@ -382,6 +383,8 @@ export const LeafyGreenSegmentedControl: React.VFC<EnumSpruceWidgetProps> = ({
         value={value}
         onChange={onChange}
         aria-controls={ariaControls?.join(" ")}
+        // @ts-expect-error
+        size={sizeVariant}
```

Comment:
You could use a type assertion instead after importing `SegmentedControlProps`

```suggestion
        size={sizeVariant as SegmentedControlProps["size"]}
```

Reply:
thank you typescript queen 🫶